source/basic.tex

```diff
@@ -4059,8 +4058,8 @@
 \tcode{void}~(\ref{basic.types}) has three corresponding cv-qualified
 versions of its type: a \defn{const-qualified} version, a
 \defn{volatile-qualified} version, and a
-\defn{const-volatile-qualified} version. The term
-\term{object type}~(\ref{intro.object}) includes the cv-qualifiers
+\defn{const-volatile-qualified} version. An
+object's type~(\ref{intro.object}) includes the cv-qualifiers
```

Comment:
Uh, our definition of \grammarterm has serious trouble handling
\grammarterm{cv-qualifier}s
(plural "s" after singular grammar non-terminal). There's lots of whitespace in between. That's also visible in 7.1.1 [dcl.stc] p5 (the note at the end; twice).